build-tools/packages/build-cli/package.json

```diff
@@ -23,8 +23,7 @@
   ],
   "scripts": {
     "build": "npm run build:genver && concurrently npm:build:compile npm:lint && npm run build:docs",
-    "build:clean": "npm run clean",
-    "build:compile": "npm run build:clean && npm run tsc && npm run build:copy",
+    "build:compile": "npm run tsc && npm run build:copy",
```

Comment:
Including "clean" in the compile steps breaks the incremental build heuristics.

Typically manifests as API-Extractor not being able to locate 'index.d.ts'.

Reply:
This isn't a safe change for build-cli. It needs to remove some generated files (most notably the manifest) or development gets really weird because the typical dev flow recompiles stuff on the fly -- but not the manifest. So if it's there, commands can appear to go missing etc. because the code and manifest are outdated.